lib/functions/transfer.js

```diff
@@ -75,25 +76,10 @@ class Transfer extends AsyncGeneratorFunction {
                     throw Error(`'contentRange.length' too large, not supported yet: ${transferPart.source.url}`);
                 }
 
-                console.log(`Transferred content range: ${contentRange}`);
```

Comment:
nth: keep this log message

Reply:
THose logs were debug logs I added in an earlier version to trace the issue - do we really still need them?

Reply:
I'm not sure we need them, but I added a `debug` logger to log these statements:
https://github.com/adobe/node-httptransfer/blob/master/lib/logger.js

I don't think we need all the log statements, but some of them might be useful. It would be good to make sure that test code is not committed to master/main though.

Reply:
Which one do we want though? Because I found the current logs to be quite verbose for up and download, making our logs crowded. We could keep how much has been transferred, but I wouldn't keep much more to have less crowded logs for up/downloads.